src/lib/keccak.unit-test.ts

```diff
@@ -1,106 +1,136 @@
 import { Field } from './field.js';
```

Comment:
now that we have a full end to end functional Keccak, could you also add that (and some of the other variants, where you see fit) to the verification key regression tests?